coderd/users_test.go

```diff
@@ -387,6 +390,7 @@ func TestPostUsers(t *testing.T) {
 			Password:       "testing",
 		})
 		require.NoError(t, err)
+		assert.Len(t, auditor.AuditLogs, 1)
```

Comment:
would be good to assert more more about the log, like action, resource type & name, status, etc.

same goes for the other tests.

Reply:
I think in the API tests it's really only important that we assert the audit log is being generated and the action. The other fields are testable elsewhere.

Reply:
Where are you suggesting we test the other fields, and which ones do you think will be covered?

I feel like this is an ideal place to test that user API calls generate audit logs with a resource type user, for example. Or that a successful API call results in an audit log recording status 200.

Also, that RequestID is non-empty.

Reply:
Resource Type/ID/Target: Derived from the generic `InitRequest` is called with, so it is type enforced to be correct
Status code: Logic is all in the audit package, so it can just be tested there instead
Request ID: Logic is also in the audit package